python/spglib/spglib.py

```diff
@@ -35,7 +35,10 @@
 
 from __future__ import annotations
 
+import dataclasses
 import warnings
+from abc import ABC
+from collections.abc import Mapping
```

Comment:
```suggestion
if sys.version_info < (3, 9):
    from typing import Mapping
else:
    from collections.abc import Mapping
```

Python 3.8 support is annoying. Maybe this does the trick?